src/Components/CustomMenu/CustomMenu.scss

```diff
@@ -0,0 +1,8 @@
+@import '../../theme.scss';
+
```

Comment:
Not sure this file is in use

Reply:
I have removed the file altogether, it was for customizing the menu but we are now using the material ui approach.